js/utils/modalManager.js

```diff
@@ -57,6 +59,23 @@ export function launchSettingsModal(modalOptions = {}) {
   return aboutModal;
 }
 
+export function launchPurchaseModal(modalOptions = {}) {
```

Comment:
Ah, ok. I thought it was better to run everything through the modal manager, so all modals were handled the same way. That way we could always handle positioning and layering the same.